mvnw

```diff
@@ -199,6 +203,85 @@ if [ -z "$BASE_DIR" ]; then
   exit 1;
 fi
 
+##########################################################################################
+# Extension to allow automatically downloading the maven-wrapper.jar from Maven-central
```

Comment:
Maybe we should remove the jar then?

Reply:
I don't have strong personal opinion.
If the maven-wrapper.jar is not present the version which should be downloaded is specified by `wrapperUrl` in `maven-wrapper.properties`.

Reply:
What is recommended by maven wrapper? We download maven anyway, so downloading one more jar is not a big deal.

Reply:
Ok. Let's keep the verbose copy. It is part of the maven wrapper tool, so we should not change it. It is kind of accidental that it is bash.